src/hooks/usePersonalData/usePersonalData.tsx

```diff
@@ -77,11 +88,25 @@ const usePersonalData = () => {
     },
     [dispatch]
   )
-  return { state, updateWatchedVideos, updateChannelFollowing }
+
+  const updateRecentSearches = useCallback(
```

Comment:
The purpose of this function isn't to update recent searches though, right? It's a side effect but the main purpose is to add a recent search, so let's make the function name communicate that